recipes/libmatio/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+./autogen.sh
+./configure --quiet --enable-shared --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+eval ${LIBRARY_SEARCH_VAR}=$PREFIX/lib make check
```

Comment:
So, I pointed to that line, but `LIBRARY_SEARCH_VAR` is not globally defined. It should be defined to either `DYLD_FALLBACK_LIBRARY_PATH` on Mac or `LD_LIBRARY_PATH` on Linux. Take a look at [this `build.sh`]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh ) for more details.